deploy/vagrant/conf/gce.yml.template

```diff
@@ -0,0 +1,61 @@
+# Setting up your Google Cloud Project
+#
+# The Project ID for your Google Cloud Platform account.
+Google_Cloud_Project_ID:
+
+# The Client Email address for your Service Account.
+Serice_Acount_Email_Address:
+
+# The location of the JSON private key file matching your Service Account.
+json_Key_Location: 
+
+
+#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
+#
+# This is the path to SSH keys. The easiest way to setup GCE keys is using the 
+# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to 
+# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
+# directly or manually manage keys.
+Key_Path: "~/.ssh/google_compute_engine"
+
+# This is the username associated with the keys above.
+SSH_Username: "gce-user"
+
+
+# Configuring virtual machines
+#
+#Boolean whether to enable preemptibility. Default is false.
+Preemptible: false
+
+# See the web docs (https://cloud.google.com/compute/docs/machine-types) or 
+# CLI (gcloud compute machine-types list) for a list of available machine types.
+# When using spark, we recommend "n1-highmem-2", compile spark needs 2G memory at least
+Machine_Type: "n1-standard-1"
+
+# Scopes
+# An array of OAuth2 account scopes for services that the instance will have access to. 
+# Those can be both full API scopes, just endpoint aliases (the part after ...auth/), 
+# and gcloud utility aliases, 
+# for example: ['storage-full', 'bigquery', 'https://www.googleapis.com/auth/compute'].
```

Comment:
Can you please add the documentation/link about auth scopes, to clarify how OAuth/Scopes work?